plugins/catalog/src/components/EntityLinksCard/EntityLinksCard.tsx

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity } from '@backstage/catalog-model';
+import { InfoCard } from '@backstage/core';
+import {
+  Grid,
+  GridList,
+  GridListTile,
+  Link,
```

Comment:
`Link` from `@backstage/core`?

Reply:
From my understanding the backstage version only handles routing via the `to` prop. I'll have another look. Might be nice to have it support `href` if it doesn't already.

Reply:
It supports external links, not just routes